addons/survey/models/survey_survey.py

```diff
@@ -984,7 +996,7 @@ def _prepare_leaderboard_values(self):
             'id',
             'nickname',
             'scoring_total',
-        ], limit=15, order="scoring_total desc")
+        ], order="scoring_total desc")
```

Comment:
Well, not a fan ... then maybe it should have pages, but if we have long-running surveys, not sure we should fetch 20K answers. Would vote for a limit parameter, set to *some number you like*, and if we feel it some pagination (but hey).

Reply:
Pagination would be a pain as we need to be sure to have the 15 old first and new first scores in the first page for the animation, so setting some number I like (and PO approved).